Sources/Pokepay/BankAPI/Account/GetAccountCoupons.swift

```diff
@@ -0,0 +1,46 @@
+import APIKit
+
+public extension BankAPI.Account {
+    struct GetCoupons: BankRequest {
```

Comment:
BankAPI.Account.GetCoupons
BankAPI.PrivateMoney.GetCoupons

maybe a little confusing. because in general the "module name" will not be written in the codes

Reply:
As the function [public func send<T: APIKit.Request>(_ request: T, handler: @escaping (Result<T.Response, PokepayError>) -> Void) {](https://github.com/pokepay/ios-sdk/blob/0ddd6492b78317079296f27d713eb3b4f35f2993/Sources/Pokepay/Pokepay.swift#L62)
We must use the whole name, like `BankAPI.Account.GetCoupons` or `BankAPI.PrivateMoney.GetCoupons` , for example,  [mclient.send(BankAPI.Cashtray.Get(id: id))](https://github.com/pokepay/ios-sdk/blob/0ddd6492b78317079296f27d713eb3b4f35f2993/PokepayTests/PokepayTests.swift#L219)

Reply:
if this is forced its ok. but I think it could be written like

```
import BankAPI.Account
...
mclient.send(GetCoupons) { result in
```